test/output-tests/run_extensions.expected

```diff
@@ -2430,48 +2410,48 @@ run_extensions.at-4380-test_stdio.c:12.6-12.7:
 
 Considering: import/gnucobol/tests/testsuite.src/run_extensions.at:4454:0
 Considering: import/gnucobol/tests/testsuite.src/run_extensions.at:4597:0
-run_extensions.at-4597-prog.cob:50.7-50.12:
+run_extensions.at-4597-prog.cob:50.32-50.33:
   47              DISPLAY 'Program is stopping'
   48              STOP RUN
   49              .
   50 >        ENTRY 'ExtProc-internal2'.
-----          ^^^^^
+----                                   ^
   51              DISPLAY 'Exit procedure from ' FUNCTION MODULE-ID ()
   52              SET Ext-Proc-Address TO ENTRY 'ExtProc'
 >> Error: Invalid syntax
 
-run_extensions.at-4597-prog.cob:80.7-80.12:
+run_extensions.at-4597-prog.cob:50.32:
+  47              DISPLAY 'Program is stopping'
+  48              STOP RUN
+  49              .
+  50 >        ENTRY 'ExtProc-internal2'.
```

Comment:
We still have an issue here, right? Maybe we need to tweak precedence of `USING` over `ENTRY`.

Reply:
I'm not sure to understand

Reply:
Are you talking about this? 

```cobol
SET Ext-Proc-Address TO ENTRY 'ExtProc'
```

I expected it would be fixed by merging the fix on the `SET` keyword.

Reply:
Here we have a syntax error on `ENTRY <aphanum>.`, which I guess is supposed to be handled by this PR.